Gemfile

```diff
@@ -39,11 +40,10 @@ group :development do
 end
 
 group :assets do
-  gem "therubyracer", require: false
   gem 'coffee-rails' #, '~> 3.2.1'
   gem 'uglifier' #, '>= 1.0.3'
   gem 'underscore-rails'
-  gem 'backbone-on-rails'
+  # gem 'backbone-on-rails'   # This gem has been yanked.
```

Comment:
Did you end up just committing backbone.js to into the vendor directory? If so, this line can be removed.